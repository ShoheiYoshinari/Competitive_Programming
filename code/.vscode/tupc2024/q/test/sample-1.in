4
3
1 4
3 7
6 7
2
1 1000000000
1 1000000000
2
1 2
2 3
4
20 25
3 22
3 23
3 24
