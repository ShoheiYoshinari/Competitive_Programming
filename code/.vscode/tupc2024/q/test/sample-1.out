2
999999999
0
43
